src/main/java/org/codehaus/mojo/buildhelper/RemoveLocalArtifactMojo.java

```diff
@@ -113,4 +124,23 @@ public void execute()
             }
         }
     }
+
+    static void modifyMetadataFile( MavenProject mavenProject, File metadataFile ) throws MojoFailureException {
+		if ( metadataFile.exists() ) {
+			try {
+		    	Reader reader = ReaderFactory.newXmlReader( metadataFile );
```

Comment:
Resource leak ... readers and writers should be closed: see https://programming.guide/java/try-with-resources.html for an example.

Reply:
I could fix this, but there does not seem to be any activity for merging PRs here. I have already forgotten my personal usecase here.

Reply:
you could ping me, @olamy or @hboutemy.
Best approach is to post to the mailing list (google groups).